### DISCUSSION TITLE
Absolute import Next 10 doesn't work for me

### DISCUSSION THREAD

**Discussion by matthijn** [vercel/next.js#23685] [Help]
I don't know how I could have done this wrong, so I'm asking this question here. I'm trying to add absolute path imports to my project. According to [this page](https://nextjs.org/docs/advanced-features/module-path-aliases). 

So, basically I should add `"baseUrl": "."` to my `tsconfig.json` and make sure that `"moduleResolution": "node"` is set. (Which was the default which I got after generating my next.js project with a `tsconfig.json` file present.)

Either way, for full disclosure, this is my `.tsconfig.json` (I did not change anything except add the `baseUrl`)

```
{
  "compilerOptions": {
    "target": "es5",
    "lib": [
      "dom",
      "dom.iterable",
      "esnext"
    ],
    "allowJs": true,
    "skipLibCheck": true,
    "strict": false,
    "forceConsistentCasingInFileNames": true,
    "noEmit": true,
    "esModuleInterop": true,
    "module": "esnext",
    "moduleResolution": "node",
    "resolveJsonModule": true,
    "isolatedModules": true,
    "jsx": "preserve"
  },
  "include": [
    "next-env.d.ts",
    "**/*.ts",
    "**/*.tsx"
  ],
  "exclude": [
    "node_modules"
  ],
  "baseUrl": "."
}
```

However doing an absolute import like

```
import { createJob } from 'domain/jobs/job'
```

in a file. Gives me `failed to compile .. Module not found: Can't resolve 'domain/jobs/job'`. Using relative imports however eg

```
import { createJob } from '../../domain/jobs/job'
```

does work fine. So I'm not sure what I'm missing here.

**Comment by wonderlandpark** [comment]
`baseUrl` option should be `compilerOptions.baseUrl`

Try like this
```json
{
  "compilerOptions": {
       "baseUrl": ".",
        ...
    }
}

**Reply by akellbl4** [reply]
Agree, `baseUrl` should be placed inside `compilerOptions `
